roles/web/templates/nginx.conf.j2

```diff
@@ -48,7 +48,7 @@ http {
 
     server {
         listen       1443;
-        server_name  {{ domain_name }};
+        server_name  {{ domain_name }} meta-{{ domain_name }};
```

Comment:
This lets us talk directly to nginx on the instance before it's able to join the LB.